### DISCUSSION TITLE
DNA hairpin loops

### DISCUSSION THREAD

**Discussion by manutamminen** [BradyAJohnston/MolecularNodes#589] [General]
Hello Brady,
Thanks so much for your work on this amazing add-on!
I was wondering, is there currently built-in functionality for simulating DNA hairpin loops or self-annealing of single-stranded DNA?
Thanks,
Manu

**Comment by BradyAJohnston** [comment]
Nothing currently to 'out of the box' support this kind of thing, but maybe at some point in the future
